src/install/log.c

```diff
@@ -201,7 +201,7 @@ _noreturn_ static void log_assert(const char *text, const char *file, int line,
 {
         static char buffer[LINE_MAX];
 
-        snprintf(buffer, sizeof(buffer), format, text, file, line, func);
+        snprintf(buffer, sizeof(buffer), format, text, file, (unsigned int)line, func);
```

Comment:
This doesn't make any sense. Either change the formats in the two functions that call `log_assert` or change the function signatures to take line as unsigned.

Reply:
Done, ty.